cluster/manifests/deletions.yaml

```diff
@@ -164,14 +164,50 @@ post_apply:
   namespace: kubenurse
   kind: Service
 {{- end }}
-
 - name: karpenter-logging-config
   kind: ConfigMap
   namespace: kube-system
+- name: karpenter-webhook
+  kind: ClusterRole
+- name: karpenter-webhook
+  kind: ClusterRoleBinding
+- name: karpenter-webhook
+  kind: ClusterRole
+- name: karpenter-webhook
+  kind: ClusterRoleBinding
+- name: karpenter-global-settings
+  kind: ConfigMap
+  namespace: kube-system
+- name: config-logging
+  kind: ConfigMap
+  namespace: kube-system
+- name: karpenter-cert
+  kind: Secret
+  namespace: kube-system
+- name: defaulting.webhook.karpenter.k8s.aws
+  kind: MutatingWebhookConfiguration
+- name: validation.webhook.karpenter.k8s.aws
+  kind: ValidatingWebhookConfiguration
+- name: validation.webhook.config.karpenter.sh
+  kind: ValidatingWebhookConfiguration
+- name: validation.webhook.karpenter.sh
+  kind: ValidatingWebhookConfiguration
```

Comment:
Are all these legacy things that should be deleted for the update?

Maybe add a comment to make it clear? :)